docs/getting-started/freqaskques.md

```diff
@@ -97,7 +97,21 @@ To get up and running with the Zowe CLI component quickly, see [Zowe CLI quick s
 
 <summary></summary>
 
-The primary prerequisites is Java on z/OS and the z/OS Management Facility enabled and configured. For a complete list of software requirements listed by component, see [System requirements](../user-guide/systemrequirements.md).
+Prerequisites vary by component used, but in most cases the primary prerequisites are Java and NodeJS on z/OS and the z/OS Management Facility enabled and configured. For a complete list of software requirements listed by component, see [System requirements](../user-guide/systemrequirements.md).
+
+</details>
+
+### What's the difference between using Zowe with or without Docker?
+
+<details class="zowe-faq">
+
+<summary></summary>
+
+There are multiple download options for Zowe server components, such as PAX and SMPE. Docker is another option, which allows you to run some Zowe server components outside of z/OS.
```

Comment:
Docker is a download option for Zowe that allows you to run certain Zowe server components outside of z/OS.